infra/helper.py

```diff
@@ -340,6 +340,10 @@ def get_parser():  # pylint: disable=too-many-statements
   coverage_parser.add_argument('--corpus-dir',
                                help='specify location of corpus'
                                ' to be used (requires --fuzz-target argument)')
+  coverage_parser.add_argument('--public',
```

Comment:
I think there was a bug there that I noticed too.

Reply:
> Actually what's this for?

I'm not sure I fully understand the question. In general the idea behind this option is to make it possible to run something like `helper.py coverage --public project` to let `helper.py` use public corpora that can be easily downloaded instead of using `gsutil`, tokens and the latest corpora. Without this patch `infra/helper.py coverage` always fails with something like
```
Traceback (most recent call last):
  File "/home/oblivion/project/oss-fuzz/infra/helper.py", line 1377, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/oblivion/project/oss-fuzz/infra/helper.py", line 190, in main
    result = coverage(args)
             ^^^^^^^^^^^^^^
  File "/home/oblivion/project/oss-fuzz/infra/helper.py", line 1077, in coverage
    if not download_corpora(args):
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oblivion/project/oss-fuzz/infra/helper.py", line 1016, in download_corpora
    if args.public:
       ^^^^^^^^^^^
AttributeError: 'Namespace' object has no attribute 'public'
```
because `coverage` calls `download_corpora` and `download_corpora` expects the `public` attribute to be there.

Reply:
See this one for reference @jonathanmetzman https://github.com/google/oss-fuzz/pull/9155

Reply:
I'd add that even though https://github.com/google/oss-fuzz/pull/9155 is useful by itself I kind of hope that it was just the first step towards bringing FI to `helper.py`. I kind of use it a lot to figure out how various fuzz targets fare (and most of those fuzz targets aren't integrated info OSS-Fuzz) so I think it would be great if it would be a bit easier to use it. At this point it needs some getting used to to even build fuzz targets: https://github.com/ossf/fuzz-introspector/issues/587#issuecomment-1329815097 but I think builds should be easy in general to make it easier to focus on improving things (or not getting them worse at least).

Reply:
> I kind of hope that it was just the first step towards bringing FI to `helper.py`

You're hope is not in vain -- this is the first step! I will try to push something before end of tomorrow that improves this.